utility/unit_of_work/actor_test.go

```diff
@@ -263,13 +263,13 @@ func TestUtilityUnitOfWork_GetUnbondingHeight(t *testing.T) {
 			var err error
 			switch actorType {
 			case coreTypes.ActorType_ACTOR_TYPE_APP:
-				unstakingBlocks, err = uow.getAppUnstakingBlocks()
+				unstakingBlocks, err = getGovParam[int64](uow, typesUtil.AppUnstakingBlocksParamName)
 			case coreTypes.ActorType_ACTOR_TYPE_FISH:
-				unstakingBlocks, err = uow.getFishermanUnstakingBlocks()
+				unstakingBlocks, err = getGovParam[int64](uow, typesUtil.FishermanUnstakingBlocksParamName)
 			case coreTypes.ActorType_ACTOR_TYPE_SERVICER:
-				unstakingBlocks, err = uow.getServicerUnstakingBlocks()
+				unstakingBlocks, err = getGovParam[int64](uow, typesUtil.ServicerUnstakingBlocksParamName)
 			case coreTypes.ActorType_ACTOR_TYPE_VAL:
-				unstakingBlocks, err = uow.getValidatorUnstakingBlocks()
+				unstakingBlocks, err = getGovParam[int64](uow, typesUtil.ValidatorUnstakingBlocksParamName)
 			default:
 				t.Fatalf("unexpected actor type %s", actorType.String())
 			}
```

Comment:
1 & 2 ignore
3 - I dont think this is necessary.
4 - Error is handled with `require.NoError(t, err)` after the switch etc

Reply:
Not valuable, but it's still interesting to learn where AI stands with regard to coding reviews.

I'll give it a couple years before it can replace us 😅